Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallDiffCommand.cs

```diff
@@ -516,21 +533,34 @@ private void AddDirectory(string dirName, CancellationToken cancellationToken)
             _modifiedFilesStatusReporter.IsActive.Value = true;
             _modifiedFilesStatusReporter.Description.Value = "Applying diffs...";
 
-            for (int i = 0; i < _diffSummary.ModifiedFiles.Length; i++)
+            var nameFiles = _pack1Meta.Files.Select(f => f.Name).Where(f => !_diffSummary.AddedFiles.Contains(f))
```

Comment:
Instead of `!_diffSummary.AddedFiles.Contains` you can do `_diffSummary.ModifiedFiles.Contains`.

Reply:
done